src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrectOpenClosedTag.java

```diff
@@ -0,0 +1,88 @@
+/*
+JavadocParagraph
+violateExecutionOnNonTightHtml = (default)false
+allowNewlineParagraph = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
+
+/**
+ * Some summary.
+ *
+ * <p>  Some Javadoc. </p>
+ */
+// violation 2 lines above '<p> tag should be placed immediately before the first word'
+public class InputJavadocParagraphIncorrectOpenClosedTag {
+
+    /**
+    * some javadoc.
+    * <p></p>
+    */
+    // violation 2 lines above '<p> tag should be preceded with an empty line.'
+    int a;
+
+    // violation 2 lines below '<p> tag should be preceded with an empty line.'
+    /**
+    * Some Javadoc.<P>
+    * Some Javadoc.<P></P>
+    */
+    // violation 2 lines above '<p> tag should be preceded with an empty line.'
+    int b;
+
+    // violation below 'Redundant <p> tag.'
+    /**<p>some javadoc.</p>
+    *
+    * Some <p>Javadoc.</p>
+    *
+    * Some <p>Javadoc.</p>
+    */
+    // violation 5 lines above 'Empty line should be followed by <p> tag on the next line.'
+    // violation 4 lines above 'Empty line should be followed by <p> tag on the next line.'
+    int c;
+
+    /**
+    * Some Summary.
+    *
+    * <p>
+    *   Some Javadoc.
```

Comment:
this might be  false-negative.
Indentation spaces should trigger a violation.

this is no violation code:
```
    * <p>
    * Some Javadoc.
```

Reply:
This code was previously suggested by you:

https://github.com/checkstyle/checkstyle/blob/619265de69dabb63264cddaa387fc524c656a0f6/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java#L213-L215

In this input file allowNewlineParagraph is set to true, based on the above code this example should not give violation if allowNewlineParagraph is set to true.

New lines are allowed, so I think this is correct code. `some javadoc` does not need to be right after `<p>`

Reply:
>* <p>
>    *   Some Javadoc.

If we give violation on this code and use `<p> should be placed immediately before text, without space` message then it would confuse users. Correct me if I'm wrong.

Reply:
Lets not be blocked by this,  we might need to move this discussion on a side to separate issue.

Reply:
please create issue